kotlinx-coroutines-core/common/src/flow/Flow.kt

```diff
@@ -177,7 +177,7 @@ import kotlin.coroutines.*
  * These implementations ensure that the context preservation property is not violated, and prevent most
  * of the developer mistakes related to concurrency, inconsistent flow dispatchers, and cancellation.
  */
-public interface Flow<out T> {
+public fun interface Flow<out T> {
```

Comment:
IMO, it's better not to make it `fun` as there will be confusion, on what to use when building flow:
1. `Flow { collector -> ... }` 
2. `flow { //this: FlowCollector ... }` 
And second is preferred, because it uses `SafeFlow` which checks `emit` context
https://github.com/Kotlin/kotlinx.coroutines/blob/6e0099d3dd3f54aa740f798359485b753a76a65c/kotlinx-coroutines-core/common/src/flow/Builders.kt#L56

Reply:
The first one is uppercase, the second one is lowercase, it should be fine?

Reply:
I mean, that they are too similar to misuse them, not to get conflict of declarations

Reply:
Alright, reverted.